src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -592,6 +607,7 @@
     CopyRepoArtifacts;
     CopyInnerBuildRestoredPackages;
     ExtractToolPackage;
+    DiscoverSymbolsArchiveFile;
```

Comment:
nit: this isn't necessary as it's not a standalone target. It's just a utility target for the `ExtractSymbolsArchiveFile` target.

```suggestion
```

Reply:
It is needed for proper ordering, my test build failed and I didn't play with it further - I'm keeping this for now. Many targets related to symbols would go away completely when we implement https://github.com/dotnet/source-build/issues/4225